lib/workos/types.rb

```diff
@@ -6,6 +6,7 @@ module WorkOS
   # so we're using Sorbet throughout this Ruby gem.
   module Types
     require_relative 'types/connection_struct'
+    require_relative 'types/list_struct'
```

Comment:
This really threw me though a loop and lost a couple hours to finding this...

Thoughts on pulling `require_all`?

https://github.com/jarmo/require_all

Reply:
Will follow up with this in a future PR to avoid merge conflicts.